phenotrex/ml/prediction.py

```diff
@@ -37,3 +49,17 @@ def predict(fasta_files=tuple(), genotype=None, classifier=None, verb=False):
     print("Identifier\tTrait present\tConfidence")
     for record, result, probability in zip(gr, preds, probas):
         print(f"{record.identifier}\t{translate_output[result]}\t{str(round(probability[result], 4))}")
+
+    if out_explain_per_sample is not None or out_explain_summary is not None:
+        sh = ShapHandler.from_clf(model)
+        fs, sv, bv = model.get_shap(gr)
```

Comment:
In general, I'd avoid lots of these two-letter variables as they tend to be incomprehensible. Here I think it's okay, as they are used in a quite compact context (and the next line explains the names).